common/test/acceptance/pages/lms/video/video.py

```diff
@@ -677,7 +677,7 @@ def state(self):
         elif 'is-ended' in current_state:
             return 'finished'
 
-    def _wait_for(self, check_func, desc, result=False, timeout=200):
+    def _wait_for(self, check_func, desc, result=False, timeout=200, try_interval=0.2):
```

Comment:
Decreased `try_interval` from 0.5 sec to 0.2 sec to be more precise with video position.